libsplinter/src/service/network/interconnect/mod.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright 2018-2020 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! The interconnect module provides the ServiceInterconnect sturct, which may be used to route
```

Comment:
s/sturct/struct/ (also "rouate" below, which I think is a duplicate of "route" at the end of this line).

Reply:
fixed